### PR TITLE
fix: non-existing Namespace displays details/list

### LINKED ISSUE #2198: Non existing namespace displaying resources
**Description**
When I provide wrong url path to namespace that doesn't exist I can go threw all resources and try to Create one

**Expected result**
Busola displays error and provides to namespace overview

**Actual result**
image
image
image

**Steps to reproduce**
1. Provide wrong url to namespace details
2. Check view and try to create some resource

**Troubleshooting**

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

- add TS to NamespaceRoutes & NotificationContext
- move useHiddenNamespaces to shared hooks
- add handler to check whether Namespace you want to interact exists, if not `IncorrectPath` component is displayed
- adjust extensionsAtom to the above handler
- Update Extensibility default value in docs

**Related issue(s)**
Closes #2198

### CODE REVIEW

**Review by akucharska — DISMISSED** [review]

File: core-ui/public/defaultConfig.yaml [hunk 1]
```diff
@@ -61,7 +61,7 @@ config:
     VISUAL_RESOURCES:
       isEnabled: true
     EXTENSIBILITY:
-      isEnabled: true
+      isEnabled: false
```
Comment (mrCherry97) [inline core-ui/public/defaultConfig.yaml:64]: We decide with V to enable it by default

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**